tests/xtimer_usleep/main.c

```diff
@@ -25,24 +27,23 @@
 #define ONE_SEC_SLEEP     (1 * US_PER_SEC)
 #define FIVE_SEC_SLEEP    (5 * US_PER_SEC)
 #define TEN_SEC_SLEEP     (10 * US_PER_SEC)
+#define T                 (9U)
```

Comment:
`#define NUM_LOOPS (9U)` ?

Reply:
Also, why 9? IMHO the test does not loose any of its credibility when doing 5 (or even 3) runs, right?!